BungieFrameWork/BFW_Source/BFW_Bink/BFW_Bink.c

```diff
@@ -13,6 +13,29 @@
 #include "Oni_Motoko.h"
 #include "Oni_Persistance.h"
 
+#if UUmPlatform == UUmPlatform_Linux
```

Comment:
In principle I concur. These definitions are merely to "get it out the door".
However, since these are only used in `static` function declarations the current 'BINK_VIDEO` define can just also cover the complete functions, instead of just the bodies.